beacon_node/network/src/sync/block_lookups/tests.rs

```diff
@@ -215,9 +207,9 @@ impl TestRig {
         }
     }
 
-    fn log(&self, msg: &str) {
-        info!(self.log, "TEST_RIG"; "msg" => msg);
-    }
+    // fn log(&self, msg: &str) {
+    //     info!(self.log, "TEST_RIG"; "msg" => msg);
+    // }
```

Comment:
I think we may as well reuse this. Something like this:
```suggestion
    fn log(msg: &str) {
        info!(msg, "TEST_RIG");
    }
```